functions/sg/deploy.sh

```diff
@@ -0,0 +1,52 @@
+# Deploy the submit_sg_workload cloud function
+# usage: bash deploy.sh TRIGGER_BUCKET_NAME
+# ex: bash deploy.sh broad-gotc-prod-storage
+
+TRIGGER_BUCKET_NAME=${1}
+TRIGGER_EVENT=${2:-"google.storage.object.finalize"}
+REGION=${3:-"us-central1"}
+
+# gotc-prod
+if [ "${TRIGGER_BUCKET_NAME}" == "broad-gotc-prod-storage" ]; then
+    GCLOUD_PROJECT="broad-gotc-prod-storage"
+    SA_EMAIL="sg-submission-fn-non-prod@broad-gotc-prod-storage.iam.gserviceaccount.com"
+    _PATH_PATTERN="^gs://broad-gotc-prod-storage/pipeline/[^/]+/[^/]+/unmapped/"
+    _WFL_URL="https://gotc-prod-wfl.gotc-prod.broadinstitute.org"
+    _CROMWELL_URL="https://cromwell-gotc-auth.gotc-prod.broadinstitute.org/"
+    _WORKLOAD_PROJECT="sg-prod"
+    _OUTPUT_BUCKET="gs://???"
+# gotc-dev
+elif [ "${TRIGGER_BUCKET_NAME}" == "???" ]; then
+    GCLOUD_PROJECT="broad-gotc-dev-storage"
+    SA_EMAIL="sg-submission-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com"
+    _WFL_URL="https://dev-wfl.gotc-dev.broadinstitute.org"
+    _CROMWELL_URL="https://cromwell-gotc-auth.gotc-dev.broadinstitute.org/"
+    _WORKLOAD_PROJECT="sg-dev"
+    _OUTPUT_BUCKET="gs://???"
+# gotc-dev -- Jack's sandbox
+elif [ "${TRIGGER_BUCKET_NAME}" == "jwarren-wfl-inputs" ]; then
+    GCLOUD_PROJECT="broad-gotc-dev-storage"
+    SA_EMAIL="sg-submission-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com"
+    _WFL_URL="https://jwarren-wfl.gotc-dev.broadinstitute.org"
+    _CROMWELL_URL="https://cromwell-gotc-auth.gotc-dev.broadinstitute.org/"
+    _WORKLOAD_PROJECT="sg-dev"
+    _OUTPUT_BUCKET="gs://jwarren-wfl-outputs"
```

Comment:
perhaps you could strip your sandbox-specific info from the PR?

Reply:
Totally, the deployment script isn't quite ready to go yet because we need to talk about infrastructure for it